playbooks/openshift-component_name/README.md

```diff
@@ -0,0 +1,3 @@
+# OpenShift Component_Name Playbooks
```

Comment:
Are these playbook and role files to be included with the final PR?

Reply:
No they are not.  I'll pull them before I take the WIP off.